libavfilter/dnn/queue.h

```diff
@@ -22,20 +22,95 @@
 #ifndef AVFILTER_DNN_QUEUE_H
 #define AVFILTER_DNN_QUEUE_H
 
+/**
+ * Linear double-ended data structure
+ */
 typedef struct Queue Queue;
 
+/**
+ * @brief Creates a Queue instance.
```

Comment:
I did a quick check, and see 'do' instead of 'does'.   
Creates --> Create, and all others below.

Reply:
Okay, got it.